common/lib/xmodule/xmodule/js/src/video/03_video_player.js

```diff
@@ -96,6 +96,16 @@ function (HTML5Video, Resizer) {
             $(window).on('unload', state.saveState);
 
             if (!state.isFlashMode() && state.speed != '1.0') {
+
+                // Work around a bug in the Youtube API that causes videos to
```

Comment:
@polesye I have some doubts the bug I linked in Jira is the same issue, and I have even more doubts there will ever be a useful update on that bug.  I updated an internal tracking bug for the issue, which I can't link here.

I will add the link anyway as it won't hurt.